include/flags.h

```diff
@@ -145,20 +145,25 @@ struct args {
     return get<T>(option).value_or(default_value);
   }
 
+  // Base case for parameter packed get_multi()
   template <class T>
-  optional<T> get(const std::vector<string_view>& options) const {
-    for (const auto& option : options) {
-      const auto value = get<T>(option);
-      if (value) {
-        return value;
-      }
-    }
+  optional<T> get_multi() const {
     return nullopt;
   }
 
-  template <class T>
-  T get(const std::vector<string_view>& options, T&& default_value) const {
-    get<T>(options).value_or(default_value);
+  template <class T, class... Opts>
+  optional<T> get_multi(const string_view& option, Opts... options) const {
+    const auto val = get<T>(option);
+    if (val) {
```

Comment:
This doesn't match the code style of the rest of the library. You can see line 98 on MASTER for an example: `if (const auto view = get_value(options, option)) {`